src/features/commands.ts

```diff
@@ -73,40 +73,6 @@ const commandsList: Command[] = [
       });
     },
   },
-  {
-    words: [`!contribution`],
```

Comment:
👀 intentional?

Reply:
👀  no, good catch not sure how I missed that even after checking. Will fix it asap